hack/make-rules/test-cmd-util.sh

```diff
@@ -221,6 +221,52 @@ setup() {
   kube::log::status "Setup complete"
 }
 
+########################################################
+# Kubectl version (--short, --client, --output) #
+########################################################
+run_kubectl_version_tests() {
+  kube::log::status "Testing kubectl version"
+
+  # create version files, one for the client, one for the server.
+  # these are the files we will use to ensure that the remainder output is correct
+  kube::test::version::object_to_file "Client" "" "client_version_test"
+  kube::test::version::object_to_file "Server" "" "server_version_test"
```

Comment:
expected output would change for each version, including testing on different machines. Any ideas on how to handle this would be awesome?